Release sage 2.1.0 (#420)
openforcefield/qca-dataset-submission: submissions/2025-05-22-OpenFF-Sage-2.1.0-Optimization-Training-Dataset-v1.0/README.md

# OpenFF Sage 2.1.0 Optimization Training Dataset v1.0

## Description

> A quantum chemical (QC) dataset of single point energies and properties was curated to train the 
> [OpenFF 2.1.0 Sage](https://github.com/openforcefield/sage-2.1.0/) forcefield, with reparametrized Lennard-Jones (LJ) 
> and valence parameters. This QC dataset with the OpenFF default level of theory, B3LYP-D3BJ/DZVP, is used to train Sage parameters. 
> This is the complete optimization dataset includes calculations used for training OpenFF 2.0.0 Sage in addition to other, consisting 
> of the following datasets:

Reviewer: ```suggestion
A quantum chemical (QC) dataset of optimization targets was generated at the OpenFF default level of theory, B3LYP-D3BJ/DZVP, and curated to train the 
valence parameters of the [OpenFF 2.1.0 Sage](https://github.com/openforcefield/sage-2.1.0/) forcefield. Targets were curated from the following datasets:
```
LJs weren't trained.
